index.html

```diff
@@ -7,15 +7,69 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <!-- Add a link  to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <nav>
+            <img class="logo" src="img/karma-logo.svg" alt="Karma Logo">
+            <a class="menu-burger" href="#"><img class="logo" src="img/menu-hamburger.svg" alt="menu Logo"></a>
+            <ul class="nav-wrapper">
+                <li><a class="nav-items" href="index.html">Meet Karma</a></li>
+                <li><a class="nav-items" href="#info">How it Works</a></li>
+                <li><a class="nav-items" href="#store-wrapper">Store</a></li>
+                <li><a class="nav-items" href="#blog-Wrapper">Blog</a></li>
+                <li><a class="nav-items" href="#">Help</a></li>
+                <li><a class="nav-items" href="#">Login</a></li>
+            </ul>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        </nav>
+
+        
+    </header>
+
+    <main>
+
+        <section id="info">
+            <h1>Introducing Karma</h1>
+            <p>Bring WIFI with you, everywhere you go.</p>
+            <button>Learn More</button>
+        </section>
+        
+        <br>
+        <br>
+        <br>
+
```

Comment:
```suggestion
```
It is not a good practice to have <br> to achieve a styling goal. It is better to use CSS to do the styling